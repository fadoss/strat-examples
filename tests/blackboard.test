games/blackboard

srew 1 2 3 4 5 6 7 8 9 using maxmin .
srew 1 2 3 4 5 6 7 8 9 using maxmax .
srew 1 2 3 4 5 6 7 8 9 using minmin .

