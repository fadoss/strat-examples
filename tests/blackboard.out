==========================================
srewrite in BLACKBOARD-STRAT : 1 2 3 4 5 6 7 8 9 using maxmin .

Solution 1
rewrites: 258
result NzNat: 5

No more solutions.
rewrites: 258
==========================================
srewrite in BLACKBOARD-STRAT : 1 2 3 4 5 6 7 8 9 using maxmax .

Solution 1
rewrites: 241
result NzNat: 1

No more solutions.
rewrites: 241
==========================================
srewrite in BLACKBOARD-STRAT : 1 2 3 4 5 6 7 8 9 using minmin .

Solution 1
rewrites: 241
result NzNat: 8

No more solutions.
rewrites: 241
Bye.
